internal/controller/hypervisor_controller.go

```diff
@@ -133,6 +134,19 @@ func (r *HypervisorReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		hypervisor.Status.Instances, _ = r.Libvirt.GetInstances()
 	}
 
+	// Update capabilities status.
+	var err error
+	hypervisor.Status.Capabilities, err = r.Libvirt.GetCapabilities()
+	if err != nil {
+		log.Error(err, "failed to get capabilities")
+		meta.SetStatusCondition(&hypervisor.Status.Conditions, metav1.Condition{
+			Type:    CapabilitiesClientType,
+			Status:  metav1.ConditionFalse,
+			Message: err.Error(),
+			Reason:  "CapabilitiesClientGetFailed",
+		})
```

Comment:
If there is a negative condition, there should be a positive in case that the capability has been fetched successful. Else you could see the error message from a previous run